### PR TITLE
191 background timeout service

### PR DESCRIPTION
**Pull request by lillie-dae**
### Description

Implements #191 .

Changes required to implement task cancelation requests.

### Status
**Ready**

### Types of changes
- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [x] New tests added to cover the changes.
- [x] All tests passed locally.
- [ ] [Documentation comments](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments) included/updated.

### CODE REVIEW

File: src/WorkflowManager/MonaiBackgroundService/Program.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/*
```
Comment (mocsharp) [inline src/WorkflowManager/MonaiBackgroundService/Program.cs:1]: Will this be deployed in a separate container? Any reason not to have the service as part of WM or TM?

Comment (lillie-dae) [reply src/WorkflowManager/MonaiBackgroundService/Program.cs]: No you are right this should have been called from WM.

File: src/WorkflowManager/MonaiBackgroundService/appsettings.Development.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
```
Comment (jackschofield23) [inline src/WorkflowManager/MonaiBackgroundService/appsettings.Development.json:4]: Are these needed now that it is going to be a background service?

Comment (lillie-dae) [reply src/WorkflowManager/MonaiBackgroundService/appsettings.Development.json]: removed all the appsettings.json and launchsettings.json

**Review by jackschofield23 — APPROVED** [review]

**Pull request merged**